types/meteor/meteor.d.ts

```diff
@@ -436,15 +444,15 @@ declare module 'meteor/meteor' {
          * @param id The new document's ID.
          * @param fields The fields in the new document.  If `_id` is present it is ignored.
          */
-        added(collection: string, id: string, fields: Object): void;
+        added(collection: string, id: string, fields: Record<string, any>): void;
```

Comment:
`unknown` would be a better choice here.

Reply:
Did you mean `Record<string, unknown>` or just `unknown`? I'm pretty sure it _does_ have to be record type at minimum.

Reply:
`Record<string, unknown>`, sorry for being imprecise.